catalog/ui/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.hbs

```diff
@@ -18,6 +18,14 @@
         Run
     </div>
 </div>
+<div class="query-interaction interaction-hit-count" title="Hit Count" data-help="Executes the search returning only the hit count">
+    <div class="interaction-icon">
+        <b>#</b>
+    </div>
+    <div class="interaction-text">
+        Hit Count
```

Comment:
❗️ Recommend renaming this menu item to be "Refresh Result Count". The term "hit" is less intuitive to the user.